core/src/test/java/org/apache/iceberg/catalog/CatalogTests.java

```diff
@@ -2272,7 +2280,7 @@ public void testCompleteCreateOrReplaceTransactionCreate() {
     if (!overridesRequestedLocation()) {
       assertThat(table.location())
           .as("Table location should match requested")
-          .isEqualTo("file:/tmp/ns/table");
+          .isEqualTo(baseTableLocation() + "/" + TABLE.namespace() + "/" + TABLE.name());
```

Comment:
thinking more about this, maybe we should add this code into `baseTableLocation()` and update it to take  the TableIdentifier as a parameter. The calling site would then just be `baseTableLocation(TABLE)`

Reply:
I changed the code accordingly